km_api/know_me/views.py

```diff
@@ -12,6 +12,41 @@
 from know_me import filters, models, serializers
 
 
+class EmergencyContactDetailView(generics.RetrieveUpdateDestroyAPIView):
```

Comment:
If you add the `Destroy` part, we would prefer it if there was a test for sending a `DELETE` request to the view.